Orange/widgets/data/tests/test_owfile.py

```diff
@@ -76,6 +76,23 @@ def setUp(self):
     def tearDown(self):
         dataset_dirs.pop()
 
+    def test_describe(self):
```

Comment:
You do not intend to test this exact wording, right? This test is meant to check that `get_nan_frequency_attribute` is not called for large data. Hence, explicitly test that it's not called. Do this by patching the function (like in the test bellow) and `assert_not_called` in the second case.

Also, rename the test into what it tests, e.g. `test_describe_not_calling_get_nans`.

I know that we have tests that test the exact wording and I may had been writing them, too -- or at least I merged some. I think they are a bad idea, though. A proper test for the description itself would be that it includes numbers 150 and 4 and the word Iris.